.github/workflows/feature_deploy.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     # environment:
     #   name: ${{ needs.deployment-environment.outputs.environment }}
     strategy:
-      max-parallel: 4
+      max-parallel: 2
```

Comment:
This is because it gets sad if you try to many at the same time so I've reduced it to 2 for now